### PR TITLE
Miscellaneous code simplication

### PR DESCRIPTION
**Pull request by chanseokoh**

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.sdk.ui.test/src/com/google/cloud/tools/eclipse/sdk/ui/MessageConsoleWriterOutputLineListenerTest.java [hunk 1]
```diff
@@ -41,7 +37,7 @@ public void testOnOutputLine() {
     String message = "a message";
     MessageConsoleWriterOutputLineListener listener = new MessageConsoleWriterOutputLineListener(mockConsoleStream);
     listener.onOutputLine(message);
-    verify(mockConsoleStream, times(1)).println(eq(message));
+    verify(mockConsoleStream).println(message);
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.sdk.ui.test/src/com/google/cloud/tools/eclipse/sdk/ui/MessageConsoleWriterOutputLineListenerTest.java]: `times(1)` doesn't seem amiss here?

File: plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/GoogleLoginServiceTest.java [hunk 2]
```diff
@@ -152,7 +151,7 @@ public void testGoogleLoginService_removeSavedCredentialIfNullRefreshToken() {
     when(savedOAuthData.getRefreshToken()).thenReturn(null);
 
     new GoogleLoginService(dataStore, uiFacade, loggerFacade);
-    verify(dataStore, times(1)).removeOAuthData("my-email@example.com");
+    verify(dataStore).removeOAuthData("my-email@example.com");
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/GoogleLoginServiceTest.java]: Does this seem like an inappropriate use of `times(1)`?

File: plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/GoogleLoginServiceTest.java [hunk 3]
```diff
@@ -164,7 +163,7 @@ public void testGoogleLoginService_removeSavedCredentialIfScopesChanged() {
     when(savedOAuthData.getRefreshToken()).thenReturn("fake_refresh_token");
 
     new GoogleLoginService(dataStore, uiFacade, loggerFacade);
-    verify(dataStore, times(1)).removeOAuthData("my-email@example.com");
+    verify(dataStore).removeOAuthData("my-email@example.com");
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/GoogleLoginServiceTest.java]: Does this seem like an inappropriate use of `times(1)`?

File: plugins/com.google.cloud.tools.eclipse.sdk.ui.test/src/com/google/cloud/tools/eclipse/sdk/ui/MessageConsoleWriterOutputLineListenerTest.java (hunk 1 above)
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.sdk.ui.test/src/com/google/cloud/tools/eclipse/sdk/ui/MessageConsoleWriterOutputLineListenerTest.java]: Can you elaborate?

**Comment by chanseokoh** [comment]
> Some of the `times(1)` seem like they may be appropriate, especially when multiple calls aren't idempotent.

I think you're thinking about `verify(..., atLeastOnce())`. The thing is that, `verify(...)` and `verify(..., times(1))` are exactly same: http://static.javadoc.io/org.mockito/mockito-core/2.7.17/org/mockito/Mockito.html#exact_verification

**Comment by briandealwis** [comment]
Ah I wrongly assumed `verify(N)` was `verify(N, atLeastOnce())`.  Thanks for the clarification!

**Pull request merged**